ScoutSuite/providers/aws/rules/findings/dynamodb-no-default-encryption.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Table without Server-Side Encryption at Rest Enabled",
```

Comment:
I'm thinking we want to change the wording of this. Based on what you told us last week, DDB is always encrypted no matter what. If this is not true, the data is encrypted by the AWS owned key that is not linked to our account. Maybe we can clarify that in the description or rationale?